trinity/rlp/block_body.py

```diff
@@ -0,0 +1,13 @@
+import rlp
+from rlp import sedes
+
+
+from eth.rlp.headers import BlockHeader
+from eth.rlp.transactions import BaseTransactionFields
+
+
+class BlockBody(rlp.Serializable):
```

Comment:
Instead of copying this in both packages, maybe it could go into: https://github.com/ethereum/eth-rlp

Reply:
The duplication is simply a half-step towards moving it entirely into `trinity` as the `p2p` module should no longer need this.  I don't know of another codebase other than `trinity` that needs this.